client/src/modules/assets/assets-registry.js

```diff
@@ -277,6 +277,20 @@ function AssetsRegistryController(
     vm.gridApi.core.notifyDataChange(uiGridConstants.dataChange.COLUMN);
   };
 
+  /**
+   * Show all the asset scans for this asset
+   * @param {string} uuid - the asset UUID
+   */
+  vm.showAssetScans = (asset) => {
+    $state.go('stockAssetsScans', {
+      filters : [{
+        key : 'asset_uuid',
```

Comment:
Can you add `cacheable : false` to this filter?  It will tell the filter service _not_ to remember this filter in the cache when you press the back button.  That way, a user doesn't have to manually clear it if they were just looking something up quickly.

Reply:
You might also want to add a filter on the time period to make it `allTime` or something similar.

Reply:
Added cacheable.  Did not add the period since we currently don't a period filter on the Asset registry.